src/gps/GPS.cpp

```diff
@@ -507,18 +507,28 @@ bool GPS::setup()
             delay(250);
         } else if (gnssModel == GNSS_MODEL_AG3335) {
 
-            _serial_gps->write("$PAIR066,1,0,1,0,0,1*3B"); // Enable GPS+GALILEO+NAVIC
+            _serial_gps->write("$PAIR066,1,0,1,0,0,1*3B");
+            delay(250); // Enable GPS+GALILEO+NAVIC
+
+            // Configure NMEA
+            _serial_gps->write("$PAIR062,0,1*3F");
+            delay(250); // GGA ON
+            _serial_gps->write("$PAIR062,1,0*3F");
+            delay(250); // GLL OFF
+            _serial_gps->write("$PAIR062,2,0*3C");
+            delay(250); // GSA OFF
+            _serial_gps->write("$PAIR062,3,0*3D");
+            delay(250); // GSV OFF
+            _serial_gps->write("$PAIR062,4,1*3B");
+            delay(250); // RMC ON
+            _serial_gps->write("$PAIR062,5,0*3B");
+            delay(250); // VTG OFF
+            _serial_gps->write("$PAIR062,6,0*38");
+            delay(250); // ZDA ON
+
+            _serial_gps->write("$PAIR080,1*2F");
+            delay(250); // Fitness mode!
```

Comment:
I couldn't make these commands work on the T1000E either, but noticed the CR/LF is missing. (\r\n) Even with those they show no effect. Are we sure TX is even connected on this chip?

Reply:
See comments in gnss on discourse today. I can't get any commands working either :(

Reply:
The CRLF does need yo be fixed though :)